src/docs/sphinx/dev_guide/tpl_build.rst

```diff
@@ -110,8 +110,11 @@ If you plan to install the developer tools, you should also run:
 
 .. code-block:: bash
 
-   $ brew install cppcheck doxygen
-   $ ln -s /opt/homebrew/opt/llvm@19/bin/clang-format /opt/homebrew/bin/clang-format
+   $ brew install llvm@14 cppcheck doxygen
+   $ ln -fs /opt/homebrew/opt/llvm@14/bin/clang-format /opt/homebrew/bin/clang-format
```

Comment:
Are there instructions on how to install the version of CMake we need?

Reply:
https://serac.readthedocs.io/en/task-chapman39-enzyme-macmini/sphinx/dev_guide/tpl_build.html#macos